### PR TITLE
Collapsible sidebar - Mustafe

### PR DESCRIPTION
**Pull request by Muste8**
### Changes
Give a short summary of the changes made in this PR. 
-Added button and functionality to close sidebar
-Added background image for closed sidebar look

### Link to Issue 
Link Asana ticket here!
https://app.asana.com/0/1206510424537042/1206945803599548/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 
This task took longer than I expected but I was sick this week, so there is that too

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!

https://github.com/softwareforgood/innocent_tech_app/assets/147085122/3d3e3bd1-b797-445c-8d77-cb5e463509c5

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/javascript/controllers/sidebar_controller.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["sidebar", "collapse"];
```
Comment (camckin10) [inline app/javascript/controllers/sidebar_controller.js:4]: @Muste8  
It totally makes sense that you would need to add 'collapse' to the controller! 
Was this the line changes that you discovered caused the code to function on Wed? Or what the change located in another spot of the code?

**Review by camckin10 — APPROVED** [review]
Excellent work! 
Left a question, but once you answer and resolve the question, you're good to merge!

File: app/views/layouts/_sidebar_nav.html.erb [hunk 2]
```diff
@@ -1,61 +1,74 @@
...
-    <ul class="space-y-2 font-medium">
-      <li class="flex justify-center py-2 mb-8">
-        <!--innocent_tech logo here-->
-        <img src="<%= asset_path("new_sidebar_icons/logo.png") %>" alt="innocent_tech_logo" class="h-9" />
-      </li>
-      <li>
-          <%= link_to students_path, class: "nav-tab" do %>
+
+<div data-controller="sidebar" class="relative left-0 top-0 h-screen">
+  <aside id="separator-sidebar" data-sidebar-target = "sidebar" class="z-40 -translate-x-full transition-transform sm:translate-x-0" aria-label="Sidebar">
```
Comment (Muste8) [inline app/views/layouts/_sidebar_nav.html.erb:4]: This is what I was missing to specifically target the sidebar: data-sidebar-target = "sidebar"

File: app/javascript/controllers/sidebar_controller.js (hunk 1 above)
Comment (Muste8) [reply app/javascript/controllers/sidebar_controller.js:4]:
```comment
  <aside id="separator-sidebar"` data-sidebar-target = "sidebar"` class="z-40 -translate-x-full transition-transform sm:translate-x-0" aria-label="Sidebar">
```

**Pull request merged**